components/Cards.js

```diff
@@ -20,3 +20,64 @@
 // Add a listener for click events so that when a user clicks on a card, the headline of the article is logged to the console.
 //
 // Use your function to create a card for each of the articles, and append each card to the DOM.
+axios.get("https://lambda-times-api.herokuapp.com/articles").then((res) => {
+  let articleCat = res.data.articles;
+  console.log(res.data.articles);
+
+  articleCat.bootstrap.forEach((article) => {
+    document
+      .querySelector(".cards-container")
+      .appendChild(createCards(article));
+  });
+  articleCat.javascript.forEach((article) => {
+    document
+      .querySelector(".cards-container")
+      .appendChild(createCards(article));
+  });
+  articleCat.jquery.forEach((article) => {
+    document
+      .querySelector(".cards-container")
+      .appendChild(createCards(article));
+  });
+  articleCat.node.forEach((article) => {
+    document
+      .querySelector(".cards-container")
+      .appendChild(createCards(article));
+  });
+  articleCat.technology.forEach((article) => {
+    document
+      .querySelector(".cards-container")
+      .appendChild(createCards(article));
+  });
+});
+
+function createCards(article) {
```

Comment:
I would name this `createCard` without the `s` since you're creating a single card :)